spec/factories/holding.rb

```diff
@@ -15,6 +15,8 @@
     status {}
     uuid { SecureRandom.uuid }
     gov_doc_flag { [true, false].sample }
+    delete_flag { 0 }
+    enum_chron { "" }
```

Comment:
This (`enum_chron { "" }`) was done so that holdings built with this factory don't get `nil` for the normalized enumchron fields, which become `NULL` when persisted to the database. Wouldn't normally be a problem but caused the round-trip test in `pre_load_backup_spec.rb` to fail.

Reply:
Probably fine. We should be treating nil/null/empty string as the same for enumchron-related purposes in any case.